From bdaa99630113b7cb9d5e564a5777497d7530b490 Mon Sep 17 00:00:00 2001
From: VladimirMangos <vladimir@getmangos.com>
Date: Tue, 15 Feb 2011 02:15:26 +0100
Subject: [PATCH] ScriptDev2 patch

---
 src/bindings/CMakeLists.txt |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/bindings/CMakeLists.txt b/src/bindings/CMakeLists.txt
index 13eedc4..d88ebb5 100644
--- a/src/bindings/CMakeLists.txt
+++ b/src/bindings/CMakeLists.txt
@@ -18,4 +18,4 @@
 # Clone https://github.com/mangoszero/scripts here, and uncomment the next
 # line.
 
-# add_subdirectory(scripts)
+add_subdirectory(scripts)
-- 
1.7.3.1.msysgit.0

